avocado/plugins/plugins.py

```diff
@@ -31,6 +33,19 @@ class Plugins(CLICmd):
     name = 'plugins'
     description = 'Displays plugin information'
 
+    def configure(self, parser):
+        parser = super().configure(parser)
+        help_msg = 'Will list the plugins in execution order'
```

Comment:
Will list the plugins where? Could you elaborate this help, please?

Reply:
This may be misleading because the command behavior seems to show plugins based on their default priority, but not based on the execution order.

My point is that the order can still be configured via `plugins.$type.order` configuration key.

So, I'd suggest either:

* change the message to indicate that it will show the default order of execution (their default priority), OR
* change the feature to show the *actual* execution order, consisting of both the default order (priority) and actual order configuration.

I believe the latter is a more comprehensive and definitive solution.

Reply:
yes, you are right, I will update that to respect the `plugins.$type.order` as well.